File: Python/1 - Iniciante/1010.py
# coding: UTF - 8

[a, b, c] = input().split()
[d, e, f] = input().split()
b = int(b)
c = float(c)
e = int(e)
f = float(f)

t = (b * c) + (e * f)

print('VALOR A PAGAR: R$ {:.2f}'.format(t))

File: Lua/Iniciante/1010.lua
--[Funcional]

local a1 = io.read('n')
local b1 = io.read('n')
local c1 = io.read('n')
local a2 = io.read('n')
local b2 = io.read('n')
local c2 = io.read('n')

local Total = b1 * c1 + b2 * c2

print(string.format('VALOR A PAGAR: R$ %.2f', Total))